modules/apps/ccv/types/expected_keepers.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+// StakingKeeper defines the contract expected by parent-chain ccv module.
+// StakingKeeper is responsible for keeping track of latest validator set of all baby chains
+type StakingKeeper interface {
+	GetValidatorSetChanges(chainID string) []abci.ValidatorUpdate
```

Comment:
CCV will ensure that parent chain is not validating two chains named `ethermint`. It is responsibility of CNS that Tharsis is the only entity allowed to start a chain named ethermint, for example

Reply:
I suspect it's simpler to remove dependence on CNS for identifier management for the core ccv module, and to ensure that this chainID is unique only within ccv.
A custom module could then extend/wrap this module with CNS-based identifier management. It would also:
- remove any semantic complexity as CNS evolves how it handles chainIDs
- allow modular use with other identifier services, e.g., DID, addresses, keys